diff --git a/base/BUILD.gn b/base/BUILD.gn
index 11ed4a78997370385267be56cac76d155f48a874..3109fd2761b204504a443a3c51fb83119dc40eff 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4407,6 +4407,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
