diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 9c33e558c3fbb3928eb9b54d9e033ac6b2251a80..27f329c542e024e91b4dc8a27898b5a80deb71ae 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2595,6 +2595,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
