diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index c52dbf6353ff9f632be64b46f12003d84b3092c1..405ed440872feae8c9d7cb9da0dee95248b153da 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1231,6 +1231,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
