diff --git a/components/os_crypt/sync/keychain_password_mac.mm b/components/os_crypt/sync/keychain_password_mac.mm
index 3d712cb6179a917a45f9f0f69ab7436b1ee4d728..87317453460532ffef1630660e43aa7a6e2a5fc0 100644
--- a/components/os_crypt/sync/keychain_password_mac.mm
+++ b/components/os_crypt/sync/keychain_password_mac.mm
@@ -64,12 +64,14 @@ std::string AddRandomPasswordToKeychain(const AppleKeychain& keychain,
 
 // static
 KeychainPassword::KeychainNameType& KeychainPassword::GetServiceName() {
+  BRAVE_GET_SERVICE_NAME
   static KeychainNameContainerType service_name(kDefaultServiceName);
   return *service_name;
 }
 
 // static
 KeychainPassword::KeychainNameType& KeychainPassword::GetAccountName() {
+  BRAVE_GET_ACCOUNT_NAME
   static KeychainNameContainerType account_name(kDefaultAccountName);
   return *account_name;
 }
