diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index a44c688fda237e7adc2be281373f51284119bc28..3681688a8fb9f0e9eb579ba2b6f9c2f91997ed42 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -248,6 +248,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
