diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index bde84542d6ff9df49ddf87082b5fb1f2138ae168..b3038ae80fe65d6fdd5d2b606a3e322cf836f498 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -851,6 +851,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
   if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
     search_provider_ = new SearchProvider(provider_client_.get(), this);
     providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1135,6 +1136,7 @@ void AutocompleteController::SortCullAndAnnotateResult(
     absl::optional<AutocompleteMatch> default_match_to_preserve) {
   result_.SortAndCull(input_, template_url_service_, triggered_feature_service_,
                       default_match_to_preserve);
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
 
 #if DCHECK_IS_ON()
   result_.Validate();
