diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 7e7f12866a3ce0494dc059e0b6fcb3ed0f62227d..f006e2781bc1dbe837232de956a9cba43d837b2f 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2196,6 +2196,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
