diff --git a/extensions/browser/api/web_request/extension_web_request_event_router.cc b/extensions/browser/api/web_request/extension_web_request_event_router.cc
index 52eff3243f1753f48ce9eef650b6d55ef8571aad..d273a87d77c73a45390563ff1e0ded259fc82a63 100644
--- a/extensions/browser/api/web_request/extension_web_request_event_router.cc
+++ b/extensions/browser/api/web_request/extension_web_request_event_router.cc
@@ -1168,6 +1168,9 @@ ExtensionWebRequestEventRouter::OnAuthRequired(
     const net::AuthChallengeInfo& auth_info,
     AuthCallback callback,
     net::AuthCredentials* credentials) {
+  ClearSignaled(request->id, kOnBeforeSendHeaders);
+  ClearSignaled(request->id, kOnSendHeaders);
+  ClearSignaled(request->id, kOnHeadersReceived);
   // No browser_context means that this is for authentication challenges in the
   // system context. Skip in that case. Also skip sensitive requests.
   if (!browser_context ||
