diff --git a/services/network/cookie_manager.cc b/services/network/cookie_manager.cc
index 53544022865dfc562e5e76fd839165ab55427cb3..331c13e54807a00fc2550eddbfe2ce77dd3a755c 100644
--- a/services/network/cookie_manager.cc
+++ b/services/network/cookie_manager.cc
@@ -395,6 +395,7 @@ CookieDeletionInfo DeletionFilterToInfo(mojom::CookieDeletionFilterPtr filter) {
   delete_info.name = std::move(filter->cookie_name);
   delete_info.url = std::move(filter->url);
   delete_info.host = std::move(filter->host_name);
+  BRAVE_DELETIONFILTERTOINFO
 
   switch (filter->session_control) {
     case mojom::CookieDeletionSessionControl::IGNORE_CONTROL:
