diff --git a/tools/symsrc/source_index.py b/tools/symsrc/source_index.py
index cbad0fbeb8107be1e3afb0fc905cc0f6a5154674..370643e6fc41f47ce61fe017fd2dcd048f4c3dc3 100755
--- a/tools/symsrc/source_index.py
+++ b/tools/symsrc/source_index.py
@@ -597,5 +597,6 @@ def main():
   return 0
 
 
+from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/tools/symsrc/source_index.py", globals(), locals())
 if __name__ == '__main__':
   sys.exit(main())
